src/Core/src/Platform/iOS/MauiTextView.cs

```diff
@@ -66,7 +80,18 @@ public UIColor? PlaceholderTextColor
 			set => _placeholderLabel.TextColor = value;
 		}
 
-		public TextAlignment VerticalTextAlignment { get; set; }
+		public TextAlignment VerticalTextAlignment
```

Comment:
Technically we're not supposed to add new APIs for service releases. Can we make this internal and add a TODO to make it public for .NET 10?

Reply:
Sorry, I meant `public override UITextAlignment TextAlignment` a few lines above here :)

Reply:
@jfversluis , I wasn't able to change it to internal, because it was an overridden property from UITextView. I have implemented new logic to update the position. Please let me know if you have any concerns.